src/main/java/org/usfirst/frc/team5687/robot/OI.java

```diff
@@ -87,6 +87,10 @@ public double getIntakeSpeed() {
         return Helpers.applyDeadband(joystick.getRawAxis(2), Constants.Deadbands.INTAKE_STICK);
     }
 
+    public double getArmsSpeed() {
+        return Helpers.applyDeadband(Gamepad.Axes.LEFT_TRIGGER.getNumber(), Constants.Deadbands.ARM) - Helpers.applyDeadband(Gamepad.Axes.RIGHT_TRIGGER.getNumber(), Constants.Deadbands.ARM);
```

Comment:
You need to use `gamepad.getRawAxis(Gamepad.Axes.LEFT_TRIGGER)` inside the applyDeadband call in order to get the stick value. The current code just gets the axis layout number, not it's real-time control value.